ereuse_devicehub/resources/models.py

```diff
@@ -34,3 +35,12 @@ def __init__(self, **kwargs) -> None:
         # to be able to use sorted containers
         self.created = kwargs.get('created', datetime.now(timezone.utc))
         super().__init__(**kwargs)
+
+
+def on_update_time(mapper, connection, thing_obj):
```

Comment:
Maybe a name like `update_object_timestamp` could be used to be more descriptive?

Reply:
Good name